features/cli/job.feature

```diff
@@ -26,12 +26,13 @@ Feature: job.feature
       | object_type | job |
       | object_name_or_id | pi |
     Then the step should succeed
-    Given all existing pods die with labels:
-      | app=pi |
+    And I wait for the steps to pass:
```

Comment:
Perhaps also update above delete command with `And I ensure "pi" jobs is deleted`